src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -6,6 +6,21 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Read the UTF-16 encoded JSON <see cref="System.String"/> and return the converted value.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <param name="json">The UTF-16 encoded <see cref="System.String"/>.</param>
```

Comment:
nit: JSON text to parse

https://github.com/dotnet/corefx/blob/6f970b09b51cb6a266a32242f8b4a6dc7f093740/src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs#L236